src/overlays/actors/ovl_Bg_Dblue_Movebg/z_bg_dblue_movebg.c

```diff
@@ -15,7 +17,36 @@ void BgDblueMovebg_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgDblueMovebg_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgDblueMovebg_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-#if 0
+void func_80A2A1E0(BgDblueMovebg* this, GlobalContext* globalCtx);
+void func_80A2A32C(BgDblueMovebg* this, GlobalContext* globalCtx);
+void func_80A2A444(BgDblueMovebg* this, GlobalContext* globalCtx);
+void func_80A2A670(BgDblueMovebg* this, GlobalContext* globalCtx);
+void func_80A2A688(BgDblueMovebg* this, GlobalContext* globalCtx);
+void func_80A2A714(BgDblueMovebg* this, GlobalContext* globalCtx);
+void func_80A2A7F8(BgDblueMovebg* this, GlobalContext* globalCtx);
+void func_80A2AAB8(BgDblueMovebg* this, GlobalContext* globalCtx);
+void func_80A2AED0(BgDblueMovebg* this, GlobalContext* globalCtx);
+void func_80A2B1A0(BgDblueMovebg* this, GlobalContext* globalCtx);
+void func_80A2B274(Actor* thisx, GlobalContext* globalCtx);
+void func_80A2B308(Actor* thisx, GlobalContext* globalCtx);
+
+extern Gfx D_060052B8[];
+extern Gfx D_0600CD10[];
```

Comment:
Those 2 externs seems unused, could you remove them?

Reply:
They're not unused, these are the symbols inside the non-matching functions. The asm relies on them since we can't change the disasm symbols.